src/util/Util.hpp

```diff
@@ -28,6 +38,88 @@ class Util {
         static bool hasAPMP(const spy::character::Character &character) {
             return character.getActionPoints() > 0 or character.getMovePoints() > 0;
         }
+
+        /**
+         *
+         * @tparam FSM Type of the used state machine.
+         * @param key  Key to query.
+         * @param fsm  State machine.
+         * @param gameRunning Is game currently running?
+         * @param isSpectator Is the requesting client a spectator?
+         * @param player Player number if requesting client is not a spectator
+         * @return Key value pair if request was successful, else std::nullopt.
+         */
+        template<typename FSM>
+        static auto handleMetaRequestKey(spy::network::messages::MetaInformationKey key, FSM &fsm, bool gameRunning,
+                                         bool isSpectator,
+                                         std::optional<Player> player) -> std::optional<MetaInformationPair> {
```

Comment:
Meine codestyle präferenz ist, alle argumente in einzelne zeilen zu schreiben wenn man schon zeilenumbrüche hat
(dismiss falls du anderer meinung bist, nur ein vorschlag)

Reply:
Bin da derselben Meinung, da war ich allerdings faul und habe es wohl dem Autoformat überlassen.